openghg/analyse/_scenario.py

```diff
@@ -506,9 +506,9 @@ def _check_data_is_present(self, need: Optional[Union[str, Sequence]] = None) ->
             if value is None:
                 missing.append(attr)
 
-                print(f"Must have {attr} data linked to this ModelScenario to run this function")
-                print("Include this by using the add function, with appropriate inputs:")
-                print("  ModelScenario.add_{attr}(...)")
+                logger.info(f"Must have {attr} data linked to this ModelScenario to run this function")
+                logger.info("Include this by using the add function, with appropriate inputs:")
+                logger.info("  ModelScenario.add_{attr}(...)")
 
         if missing:
             raise ValueError(f"Missing necessary {' and '.join(missing)} data.")
```

Comment:
Could this be replaced with a logging error type? Or does an explicit raise make the most sense here?

Reply:
Changed to logger.error